src/common/components/cards/card-kebab-menu-button.scss

```diff
@@ -62,4 +62,8 @@ button.kebab-menu-button {
     > div {
         justify-content: center;
     }
+
+    > span {
+        justify-content: center;
```

Comment:
this align the vertical ellipse on the kebab button (more actions button) on the cards ui

Reply:
Should the `> span` rule replace the `> div` rule?

Reply:
Good catch. Updated.